src/player.h

```diff
@@ -511,6 +513,11 @@ class Player final : public Creature, public Cylinder
 		void addConditionSuppressions(uint32_t conditions);
 		void removeConditionSuppressions(uint32_t conditions);
 
+		Reward* getReward(uint32_t rewardId, bool autoCreate);
```

Comment:
Remove ```autoCreate``` from the arguments list - there's really no point in having it there as it basically just causes the function to return nullptr if false.